### DISCUSSION TITLE
Support for armv6l (Raspberry Pi Zero)

### DISCUSSION THREAD

**Discussion by KicksTV** [rhasspy/piper#488] [Q&A] [answered]
I am looking to get this running on my Raspberry pi zero. Current releases will not work. Anyone know if it possible?

**Comment by synesthesiam** [comment, marked as answer]
I've tried, but building the onnxruntime (a hard dependency of Piper) has proven too difficult. I can't get any 32-bit build to complete successfully these days, even arm7l.

**Reply by KicksTV** [reply]
Thanks for the quick reply.
